src/coveralls.erl

```diff
@@ -338,10 +345,40 @@ replce_char_test_() ->
   , ?_assertEqual("foobarfoo", replace_char("foo\"foo", $", "bar")) %"
   ].
 
-fullpath_test_() ->
-  [ ?_assertEqual("/a/b", fullpath(["/", "a", "b"], []))
-  , ?_assertEqual("a/c", fullpath(["a", "b", "..", ".", "c"], []))
-  , ?_assertEqual("/", fullpath(["..", ".", "/"], []))
+expand_test_() ->
+  [ ?_assertEqual("/a/b", expand(["/", "a", "b"], []))
+  , ?_assertEqual("a/c", expand(["a", "b", "..", ".", "c"], []))
+  , ?_assertEqual("/", expand(["..", ".", "/"], []))
+  ].
+
+realpath_and_relative_test_() ->
```

Comment:
@markusn it brings problems on travis because we have got to change cwd(for precise testing `relative_to/1`) to somewhere like `/tmp/coverals`. See https://travis-ci.org/surik/coveralls-erl/builds/78457066